lib/monitoring/aws-rds/AuroraClusterMonitoring.ts

```diff
@@ -75,12 +74,6 @@ export class AuroraClusterMonitoring extends Monitoring {
   constructor(scope: MonitoringScope, props: AuroraClusterMonitoringProps) {
     super(scope, props);
 
-    if (!this.isServerlessCluster(props.cluster)) {
```

Comment:
not following why we don't want these (including line 167) checks

Reply:
Same as the other comment:

> This unfortunately drops the guard for non-serverless DatabaseClusters for Aurora monitoring since the constructs are now nearly indistinguishable based on their properties.